tests/misc_env/exec.py

```diff
@@ -27,7 +29,10 @@ def exec_command(node, **kwargs: Any) -> Tuple:
         CommandFailed   when there is an execution failure
     """
     out, err = node.exec_command(**kwargs)
-    return out.read().decode(), err.read().decode()
+
+    out = out.read().decode() if isinstance(out, ChannelFile) else out
```

Comment:
Can we avoid checking `ChannelFile`, instead going with `str` would be good.

Reply:
this way, we could avoid import statement also.

Reply:
In exec_command, we are doing `out.read()` which could return a `str` or `bytes` but mostly `str` in our case. As the need to perform `read` is for a base file type object, I went with this implementation... in future, if we change the return value, I didn't want to break it...